na.ino/na.ino.ino

```diff
@@ -0,0 +1,162 @@
+
+#include <SPI.h>// library for SPI communication (pre-loaded in arduino)
+#include <dht.h> //Library for dht11 temperature and humidity sensor
+int soggyLEDPin=9;
+int moistsoilLEDPin=6;
+int drysoilLEDPin=5;
+int PumpPin=3;
+int moistureSensorValue;
+int a   =800;
+int chipSelect=4;

```

Comment:
This will call all libraries needed for this program to run
will also help initialize pins to where different sensors are connected to the Arduino board